README.md

```diff
@@ -213,48 +244,57 @@ if(response.isRight()) {
     System.out.println(response.getLeft().getMessage());
 }
 ```
-
-### [Map Info](https://traveltime.com/docs/api/reference/map-info)
-Get information about currently supported countries and find out what points supported by the api.
-
-```java
-MapInfoRequest request = new MapInfoRequest();
-Either<TravelTimeError, MapInfoResponse> response = sdk.send(request);
-
-if(response.isRight()) {
-    System.out.println(response.get().getMaps().size());
-} else {
-    System.out.println(response.getLeft().getMessage());
-}
-```
-
-### [Geocoding (Search)](https://traveltime.com/docs/api/reference/geocoding-search)
-Match a query string to geographic coordinates.
-
-Body attributes:
-* query: A query to geocode. Can be an address, a postcode or a venue.
-* within_country: Only return the results that are within the specified country.
-  If no results are found it will return the country itself. Format:ISO 3166-1 alpha-2 or alpha-3
+### [Time Filter (Fast)](https://docs.traveltime.com/api/reference/time-filter-fast)
+A very fast version of `time_filter()`. However, the request parameters are much more limited. Currently only supports UK and Ireland.
```

Comment:
there is no pascal case in java and no such concept of time_filter function as you build requests and send them on via a generic method. I would just replace with `Time Filter`